english/escape-to-the-wilderness/wilderness.md

```diff
@@ -0,0 +1,26 @@
+You decide to escape the city for a 
+weekend and head out to the wilderness.
+
+With your bag all packed with camping
+gear, you make your way toward the mountains.
+
+The hike to a campsite was relatively 
+easy so you settle down in a sleeping bag 
+next to the campfire, and drift off to sleep. 
+
+You awake to find you're being dragged across
+the forest floor. But by who or what?
+
+Perhaps a [bear](../bear/bear.md) got a hold of you.
+
+In a moment of panic, you:
+
+[sing](../sing/sing.md)
+
+convince yourself you're [dreaming](../dream/dream.md)
+
+[meditate](../meditate/meditate.md)
+
+make a [haiku](../write-a-hiku/hiku.md) to stay calm
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.